code/_onclick/hud/action_button.dm

```diff
@@ -1,65 +1,55 @@
+// Todo:
+// Add proper garbage collection
+// Cry yourself to sleep over the existence of hud code
```

Comment:
is this still needed to be done, or is the pr ready now?

Reply:
Reasonably sure it's done, will test it tho

Reply:
Did some testing, I think we're good now?